### PR TITLE
Portfolio: add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by K0ppai**
In this pull request, I've implemented the following,

- Add linters file
- Use professional gitflow and descriptive commits
- Use Flexbox in CSS
- Use semantic html5 tags
- Use responsive values like percentages instead of pixels
- Use CSS grid
- Add button interactions
- Use HTML form and input tags
- Validate user inputs

### CODE REVIEW

**Review by bobb-Rob — COMMENTED** [review]
Hi @K0ppai, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that lacks commits from your pair programming partner. Please ensure your coding partner(s) make significant contributions to this project. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 1]
```diff
@@ -227,5 +227,39 @@ <h3 class="heading">About me</h3>
...
+  <div class="contact-bg">
+    <div class="contact-me-container">
+      <div class="cm-heading">
+        <h4>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h4>
+      </div>
+      <form action="https://formspree.io/f/moqzrddp" method="post" class="type-in">
+        <input type="text" alt="" placeholder="Full Name" id="name" name="name" maxlength="30" required>
+        <input type="email" id="email" name="email" placeholder="Email address" required>
+        <textarea name="msg" id="msg" cols="11" rows="5" placeholder="Hey I’d like to connect to..." maxlength="500" required></textarea>
+        <button type="submit">Get in touch</button>
```
Comment (stanleySimeon) [inline index.html]: Kindly I suggest you implement the button interactions (enable, hover, etc.) to make your design match the requirements and be more professional. 😄

File: index.html [hunk 2]
```diff
@@ -227,5 +227,39 @@ <h3 class="heading">About me</h3>
...
+        <p>psthu.koppai@gmail.com</p>
+      </form>
+
```
Comment (stanleySimeon) [inline index.html]: Kindly I suggest removing all unnecessary white lines in your project to make it more professional and to HTML best practices. 😄

File: index.html [hunk 3]
```diff
@@ -227,5 +227,39 @@ <h3 class="heading">About me</h3>
...
+          <li><a href="#">
+            <img src="images/medium.svg" class="medium social-media-icon">
+          </a></li>
+          <li><a href="#">
+            <img src="images/github.svg" class="github social-media-icon">
+          </a></li>
+          <li><a href="#">
+            <img src="images/angellist.svg" class="angellist social-media-icon">
+          </a></li>
+        </ul>
```
Comment (stanleySimeon) [inline index.html:259]: Kindly I suggest you follow the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-988&t=K9cgnTJBB99smpxE-0) template to make the footer as it is. 😄 

>### The Figma Design
>
>Screenshot 2023-05-02 at 14 24 06
>
>### Your Design
>
>Screenshot 2023-05-02 at 14 24 50

- [ ] Kindly make sure you add the bottom bar to follow the design

File: style.css [hunk 4]
```diff
@@ -50,7 +50,8 @@ main {
 .headline {
   display: flex;
   flex-direction: column;
-  align-content: center;
+
+  /* align-content: center; */
```
Comment (stanleySimeon) [inline style.css]: I suggest you remove the commented code in your code base to follow best practices.
Please note by following this, your project will look more professional. 😄

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @K0ppai 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# ⚠️ Invalid Code Review Request ⚠️ 

As mentioned by the previous reviewer, you have submitted a project, that lacks commits from your pair programming partner. Please ensure your coding partner(s) make significant contributions to this project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @stanleySimeon in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by shella12 — COMMENTED** [review]
Hi @K0ppai , 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] is a pair programming project with no contribution from partner. Kindly make sure your partner contributes some code into your codebse.
- [ ] Also please remove` Note` from the LICENCE section of the README. `Note `is there in the template to guide you on how you to add a LICENCE to your README. Once you have successfully added the LICENCE you can go ahead and remove it


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by shella12 — COMMENTED** [review]
Hi @K0ppai, 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a project, that **is a [pair-programming activity](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#work-and-submission-mode). Please make sure everyone of your partners have contributed to the project before submitting for another request.**

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by stanleySimeon** [comment]
> Hi @K0ppai,
> 
> # Status: Invalidated 🔴
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> ### You have submitted a project, that **is a [pair-programming activity](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#work-and-submission-mode). Please make sure everyone of your partners have contributed to the project before submitting for another request.**
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by stanleySimeon** [comment]
This student works solely on this project, kindly don't invalidate!

**Review by stanleySimeon — APPROVED** [review]
# Hi @K0ppai

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Status: Approved 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**